.changeset/dirty-mice-help.md

```diff
@@ -0,0 +1,5 @@
+---
+'@penumbra-zone/react': major
+---
+
+initial react wallet
```

Comment:
issue: trying to connect the updated minifront to the extension (i use the extension the branch `turbocrime/use-disconnect` of Prax) results in `too many rerenders in PenumbraProvider`

<img width="626" alt="image" src="https://github.com/penumbra-zone/web/assets/29180358/36bec449-afe5-4889-b2c7-e60f469d0b58">

Reply:
See the comment below about `providerConnected` variable – it solves the problem

Reply:
ah, ok, that was a late change i made and didn't test. maybe my cache thinking was incorrect